css-shared-element-transitions/index.bs

```diff
@@ -141,12 +141,11 @@ The following describes all of the [=page-transition pseudo-elements=] and their
     }
     </code></pre>
 
-    Note: The aim of the style is to size the pseudo-element to cover the
-    complete viewport.
-
-    Issue: Is this needed? For multiple independent transitions this could be
-    useful to separate which transition is represented in the particular
-    :page-transition subtree.
+    Note: This pseudo-element provides a containing block for all
+    ::page-transition-container pseudo-elements. The aim of the style is to
+    size the pseudo-element to cover the [=large-viewport-size=] and position
```

Comment:
I'm not able to get this to link to the relevant spec: https://drafts.csswg.org/css-values-4/#large-viewport-size

Reply:
You can just `<a href=` if you can't figure out the syntax

Reply:
Done. The trick was that the text shouldn't include hiphens.